### ISSUE TITLE
"Maximum call stack size exceeded" occur tauri  but not in `Boscop/web-view`

### ISSUE THREAD

**Issue by deftsp** [tauri-apps/tauri#435]
**Describe the bug**
A wasm page created by yew.  The page will be blank until right click + Reload.  Following is the Console message after the first open.  
```
wasm-bindgen: imported JS function that was not marked as `catch` threw an error: – "Maximum call stack size exceeded.↵↵Stack:↵get@http://localhost:8000/static.e31bb0bc.js:318:2…"5

wasm-bindgen: imported JS function that was not marked as `catch` threw an error: – "Maximum call stack size exceeded.↵↵Stack:↵get@http://localhost:8000/static.e31bb0bc.js:318:2…"
```

After Reload the page works and no error message showed in Console.

The save page can be opened by raw `Boscop/web-view` Firefox/Safari without any problem.

I think it's tauri's bug.

**Platform and Versions (please complete the following information):**

OS: macOS
Node: v12.16.0
NPM: 6.13.4
Yarn: 1.21.1
Rustc: 1.41.0

**Additional context**

**Stack Trace**

**Comment by deftsp** [comment]
I did some debug. It seems caused by use the method from  'tauri/api' (which is a Proxy)too early.  I'll keep on reporting.

**Comment by deftsp** [comment]
Got it. In order to import the `tauri/js`, I following a example of `yew-ccxt-webpack`, like this 
```
import tauri from 'tauri/api';

window.tauri = tauri;
```
The evil global variable conflict with the one in `tauri/api.js`.

@lucasfernog   How about renaming the window.tauri in `tauri/api` to something like `window.__tauri__` to avoid other users get the same problem?

**Comment by nothingismagick** [comment]
Thanks for digging into this @deftsp - we're discussing it right now. If you want, feel free to join us on the discord server: https://discord.gg/veQYaZw

**Comment by nothingismagick** [comment]
@lucasfernog @nklayman  - what do you two think about this idea?

**Comment by nklayman** [comment]
I don't really know how the injection/js API works, but I think it would be better to have the js API only available as an import and not as a global variable, much cleaner that way IMO.

**Comment by lucasfernog** [comment]
yeah we should do it but it's another breaking change.. although we really encourage people to use the API module instead of window.tauri

**Comment by nothingismagick** [comment]
@lucasfernog - are we going to do this?

**Comment by lucasfernog** [comment]
Closed by #703.
